digitalocean/import_digitalocean_container_registry_test.go

```diff
@@ -0,0 +1,28 @@
+package digitalocean
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDigitalOceanContainerRegistry_importBasic(t *testing.T) {
+	resourceName := "digitalocean_container_registry.foobar"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDigitalOceanTagDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckDigitalOceanContainerRegistryConfig_basic,
+			},
+
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
This test is failing for me:

```
=== RUN   TestAccDigitalOceanContainerRegistry_importBasic
--- FAIL: TestAccDigitalOceanContainerRegistry_importBasic (10.02s)
    testing.go:640: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) (len=1) {
         (string) (len=18) "docker_credentials": (string) (len=223) "{\"auths\":{\"registry.digitalocean.com\":{\"auth\":\"asupersecretauthstringthatihaveremoved\"}}}"
        }
        
        
        (map[string]string) (len=2) {
         (string) (len=18) "docker_credentials": (string) (len=223) "{\"auths\":{\"registry.digitalocean.com\":{\"auth\":\"adifferentsupersecretauthstringthatihaveremoved\"}}}",
         (string) (len=5) "write": (string) (len=5) "false"
        }
```

If the API returns a new set of credentials each time it is called, then we might need to use:

```
ImportStateVerifyIgnore: []string{"docker_credentials"},
```

Since we're using `schema.ImportStatePassthrough` the response gets passed through `resourceDigitalOceanContainerRegistryRead`. That function needs to set the value for `write` to state. It's a bit odd case since it's not a value returned by the API, but it needs to be able to refresh the entire resource with `d.Id()` only.

https://www.terraform.io/docs/extend/resources/import.html#importer-state-function